src/Services/FHIR/FhirCoverageService.php

```diff
@@ -0,0 +1,142 @@
+<?php
```

Comment:
script is missing header (license, author, copyright)

Reply:
Done